staging/src/k8s.io/cloud-provider/controllers/node/node_controller.go

```diff
@@ -572,6 +572,19 @@ func (cnc *CloudNodeController) getNodeModifiersFromCloudProvider(
 		})
 	}
 
+	if instanceMeta.AdditionalLabels != nil {
```

Comment:
Handle `nil` and empty in one pass:
```suggestion
	if len(instanceMeta.AdditionalLabels) > 0 {
```

Reply:
Nice little trick. Current code does handle empty, but this would make it a little more graceful.